src/ucp/core/ucp_mm.c

```diff
@@ -49,7 +50,7 @@ ucs_status_t ucp_mem_rereg_mds(ucp_context_h context, ucp_md_map_t reg_md_map,
         return UCS_OK; /* shortcut - no changes required */
     }
 
-    prev_num_memh = ucs_popcount(*md_map_p);
+    prev_num_memh = ucs_popcount(*md_map_p & reg_md_map);
```

Comment:
why ' & reg_md_map'?

Reply:
a number of previous memory handles is a number of `1` bits in `md_map_p` and `reg_md_map`

Reply:
ok